Shell/Shell.h

```diff
@@ -199,5 +199,5 @@ class Shell : public Core::Object {
 
 static constexpr bool is_word_character(char c)
 {
-    return c == '_' || (c <= 'Z' && c >= 'A') || (c <= 'z' && c >= 'a');
+    return c == '_' || (c <= 'Z' && c >= 'A') || (c <= 'z' && c >= 'a') || (c <= '9' && c >= '0');
```

Comment:
Perhaps we could use `isalnum(3)` or something?

Reply:
I guess we might as well, kinda sour that I lose the constexpr tho

Reply:
We could make `isalnum` & friends `constexpr` if we sprinkle some `ifdef __cplusplus`

Reply:
Or just leave this poor guy alone lol.

who cares, it's just a single function call, not in a tight loop anyway